nav2_bt_navigator/behavior_trees/navigate_to_pose_w_replanning_and_recovery.xml

```diff
@@ -8,6 +8,8 @@
   <BehaviorTree ID="MainTree">
     <RecoveryNode number_of_retries="6" name="NavigateRecovery">
       <PipelineSequence name="NavigateWithReplanning">
+        <ControllerSelector selected_controller="{selected_controller}" default_controller="FollowPath2" topic_name="controller_selector"/>
```

Comment:
"FollowPath2"?

I assume the BT changes here and in the other file are temporary / not complete. In general, I like the first-step of setting the selectors as you did in the 2 files. Set the defaults to `GridBased` `FollowPath`, then use those blackboard entries in the FollowPath / ComputePathToPose BT nodes! 

Just make sure to update all the relevant BTs

Reply:
> Just make sure to update all the relevant BTs

You mean the docs right?